lib/__tests__/findUndefinedIdentifiers-test.js

```diff
@@ -164,6 +164,22 @@ it('knows about objects', () => {
   ).toEqual(new Set(['foo', 'uuid']));
 });
 
+it('knows about identifiers declared after usage', () => {
+  expect(
+    findUndefinedIdentifiers(
+      parse(
+        `
+    function modifyAbc() {
+      abc.def = 4;
+    }
+
+    export const abc = {};
+  `,
```

Comment:
I think it is intentional because they don't want to modify the contents of the strings.